qtws.cpp

```diff
@@ -8,6 +8,8 @@
 #include <QRegExp>
 #include <QStringList>
 
+#include <iostream>
```

Comment:
Is this actually necessary? Can we remove it?

Reply:
Sure. My mistake.